### ISSUE TITLE
install.command fails

### ISSUE THREAD

**Issue by matthewadams** [subethaedit/SubEthaEdit#39]
When I copy & paste the command listed in the Advanced settings to install `see`, I get the following error:

```
$ sudo /Applications/SubEthaEdit.app/Contents/SharedSupport/bin/install.command
Password:
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- FileUtils (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Applications/SubEthaEdit.app/Contents/SharedSupport/bin/install.command:2:in `<main>'
```

I sort of feel like this should work out of the box...

**Comment by monkeydom** [comment]
It should work out of the box. 

Can you provide your system and version? (easiest done by Help->Report an Issue… and then copy the Environment section)

Oh, I have an idea. can you test this:
> `irb`
> `require ‘FileUtils'`

which should give you the same error. And then:
> `require 'fileutils'`

Which should work. If so, I have a fix. You have a case sensitive file system, yes?

**Comment by A-G-F** [comment]
I had the same problem on 2 machines, OS X 10.11.6 with ruby-1.9.3-p551 and latest OS X 13 with system ruby.
As the install.command uses ruby, both versions seem to be too old.
With RVM installed, this is a solution:

```
Last login: Mon Jan  7 19:02:17 on ttys000
Computer:~ $ cd /Applications/SubEthaEdit.app/Contents/SharedSupport/bin
Computer:bin $ sudo sh install.command
Password:
install.command: line 2: require: command not found
install.command: line 3: require: command not found
install.command: line 5: syntax error near unexpected token `('
install.command: line 5: `expandedInstallScriptPath = File.expand_path(__dir__)'

Computer:bin $ sudo /Applications/SubEthaEdit.app/Contents/SharedSupport/bin/install.command
/Applications/SubEthaEdit.app/Contents/SharedSupport/bin/install.command:5:in `<main>': undefined local variable or method `__dir__' for main:Object (NameError)

Computer:bin $ rvm list
   jruby-9.2.5.0 [ x86_64 ]
=* ruby-1.9.3-p551 [ x86_64 ]

# => - current
# =* - current && default
#  * - default

Computer:bin $ rvm use jruby
Using /Users/***/.rvm/gems/jruby-9.2.5.0

Computer:bin $ sudo /Applications/SubEthaEdit.app/Contents/SharedSupport/bin/install.command
'see' tool will be linked at "/usr/local/bin/see" to /Applications/SubEthaEdit.app
'see' man page be linked at "/usr/local/share/man/man1/see.1" to /Applications/SubEthaEdit.app
"Authentication" and "Terminal Here" Apple Scripts will be copied to 
"/Users/***/Library/Application Scripts/de.codingmonkeys.SubEthaEdit.MacFULL"

Continue? (y/n) 
Done.
Computer:bin $ 

```

**Comment by monkeydom** [comment]
The case issue has been resolved with Issue #64
